examples/x86_64.swiftinterface

```diff
@@ -0,0 +1,511 @@
+// swift-interface-format-version: 1.0
```

Comment:
Generally, it seems curious that each endpoint inconsistently features the following:
`FooOptions`
`FooRequest`
`FooResult`

Options always exists, but the request exists infrequently and response even less frequently. You might want to look at this inconsistency and question whether it is what you want.